app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>List of all posts by user</h1>
\ No newline at end of file
+<div class="main_container">
+    <div class="profile">
+        <div class="photos">
+            <ul class="photo"> 
+               <li> <img src="<%= @user.photo %>"> </li>
+            </ul>
+        </div>
+        <div class="names_posts">
+          <ul class="name_post"> 
+              <li class="list_name"> <%= @user.name %> </li>
+              <li class="list_countpost">Number of post: <%= @user.post_counter %> </li>
+           </ul>
+        </div>
+    </div>
+
+    <div class="post">
+        <% @user.posts.each do |post| %>
+        <ul class="post_info width"> 
+            <h3 class="title"><%= post.title%> #<%= post.id%></h3>
+            <li class="bio_content"> <%= post.text %> </li>
```

Comment:
- [ ] Kindly take note that for good user experience you ought to use the `link` element that will redirects users to the `post_index `page. 👍 I was only able to access the page by typing the `url `manually. ℹ️ 
- [ ] For the sake of semantics, I think the h3 element supposed not to be in the unordered list `ul`. ℹ️